appconfiguration/feature-management-sample/pom.xml

```diff
@@ -10,10 +10,10 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>feature-management-sample</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:feature-management-sample;dependency} -->
-  <name>Feature Management Sample</name>
+  <artifactId>feature-management-sample</artifactId>
+  <version>1.0.0</version>
+  <name>Manage Features and Get Configurations From App Configuration</name>
```

Comment:
And

Reply:
The principle here is 
1. keep Azure terminology, like Cosmos DB...
1. refer to https://capitalizemytitle.com/
2. keep `with`, `and`, `to` the same case 
3. ...